README.md

```diff
@@ -1614,6 +1615,162 @@ authClient.tokenManager.off('renewed');
 authClient.tokenManager.off('renewed', myRenewedCallback);
 ```
 
+## Using the library in Node as an Authentication SDK
+
+> :warning: Beta alert! This library is in beta.
+
+You can use this library on server side in your Node application as an Authentication SDK. It can only be used in this way for communicating with the [Authentication API](https://developer.okta.com/docs/api/resources/authn), **not** to implement an OIDC flow.
+
+To include this library in your project, you can follow the instruction in the [Getting started](#getting-started) section.
+
+### Configuration
+
+You only need to set the `url` for your Okta Domain:
+
+```javascript
+var OktaAuth = require('@okta/okta-auth-js');
+
+var config = {
+  // The URL for your Okta organization
+  url: 'https://{yourOktaDomain}'
+};
+
+var authClient = new OktaAuth(config);
+```
+
+### API Reference
+
+Since the Node library can be used only for the Authentication flow, it implements only by a subset of okta-auth-js APIs:
+
+* [signIn](#signinoptions-in-node)
+* [forgotPassword](#forgotpasswordoptions-in-node)
+* [unlockAccount](#unlockaccountoptions-in-node)
+* [verifyRecoveryToken](#verifyrecoverytokenoptions-in-node)
+* [tx.resume](#txresume)
+* [tx.exists](#txexists)
+* [transaction.status](#transactionstatus)
+  * [LOCKED_OUT](#locked_out)
+  * [PASSWORD_EXPIRED](#password_expired)
+  * [PASSWORD_RESET](#password_reset)
+  * [PASSWORD_WARN](#password_warn)
+  * [RECOVERY](#recovery)
+  * [RECOVERY_CHALLENGE](#recovery_challenge)
+  * [MFA_ENROLL](#mfa_enroll)
+  * [MFA_ENROLL_ACTIVATE](#mfa_enroll_activate)
+  * [MFA_REQUIRED](#mfa_required)
+  * [MFA_CHALLENGE](#mfa_challenge)
+  * [SUCCESS](#success)
+
+### `signIn(options) in Node`
+
+The goal of an authentication flow is to authenticate a User. The flow is started using `signIn`.
+
+* `username` - User’s non-qualified short-name (e.g. dade.murphy) or unique fully-qualified login (e.g dade.murphy@example.com)
+* `password` - The password of the user
+
+```javascript
+authClient.signIn({
+  username: 'some-username',
+  password: 'some-password'
+})
+.then(function(transaction) {
+  if (transaction.status === 'SUCCESS') {
+    // Redirect to your success page
+    res.redirect('https://www.yoursuccesspage.com');
```

Comment:
@nbarbettini Do we want to mention here that you also get a `sessionToken` which you can use with https://github.com/okta/okta-sdk-nodejs#sessions?

Reply:
+1